integration/spark/spark3/src/main/java/io/openlineage/spark3/agent/lifecycle/plan/catalog/IcebergHandler.java

```diff
@@ -163,6 +163,9 @@ private Optional<DatasetIdentifier> getSymlinkIdentifier(
     } else if ("glue".equals(catalogType)) {
       log.debug("Getting symlink for glue");
       value = getGlueIdentifier(tableName, session);
+    } else if ("jdbc".equals(catalogType)) {
```

Comment:
I hope someone will find some time extract this into several `IcebergWhateverHandler` classes.

Reply:
Yup, that code isn't the prettiest...